src/models/document/document-content.ts

```diff
@@ -882,6 +905,26 @@ export const DocumentContentModel = types
           });
         }
         return tileInfo;
+      },
+      duplicateTiles(tiles: IDragTileItem[]) {
+        const rowIndex = self.getRowAfterTiles(tiles);
+        const results = self.copyTilesIntoNewRows(tiles, rowIndex);
+
+        // Increment default titles when necessary
```

Comment:
Does dropping tiles do this too? No need to refactor so it is consistent, I'm just curious at this point.

Reply:
I don't think names are automatically changed anywhere else in CLUE right now, so this is a first. Getting this to work was actually one of the most time consuming parts of this story.